include/private/gcconfig.h

```diff
@@ -25,6 +25,8 @@
 #ifndef GCCONFIG_H
 #define GCCONFIG_H
 
+#define MANUAL_VDB
```

Comment:
I put this here, but I'm wondering, do we have a better place to inject custom defines into Boehm?

Reply:
Is this for our mono build or for our il2cpp build?

Reply:
Currently I'm making this work on mono, il2cpp will follow later (but we will want it in both).